src/base/utils/string.h

```diff
@@ -49,6 +47,14 @@ namespace Utils
         bool naturalCompareCaseInsensitive(const QString &left, const QString &right);
 
         QString wildcardToRegex(const QString &pattern);
+
+        template <typename T>
+        T unquote(const T &str)
+        {
+            if (str.startsWith('\"') && str.endsWith('\"'))
```

Comment:
>Don't see size check in your function. Why?

The only questionable input `"\""`, I've added a comment for it, reason is in previous [post ]
Are there any other exceptions?

[post ]: https://github.com/qbittorrent/qBittorrent/pull/6834#discussion_r118166847